tests/functional/graph_selection/test_graph_selection.py

```diff
@@ -0,0 +1,358 @@
+import os
+import json
+import pytest
+
+from dbt.tests.util import run_dbt
+from dbt.tests.tables import TableComparison
+from tests.functional.graph_selection.fixtures import models, seeds, create_tables  # noqa
+
+
+selectors_yml = """
+            selectors:
+            - name: bi_selector
+              description: This is a BI selector
+              definition:
+                method: tag
+                value: bi
+        """
+
+
+@pytest.fixture
+def selectors():
+    return selectors_yml
+
+
+def assert_correct_schemas(project, table_comp):
+    with table_comp.get_connection():
+        exists = project.adapter.check_schema_exists(project.database, project.test_schema)
+        assert exists
+
+        schema = project.test_schema + "_and_then"
+        exists = project.adapter.check_schema_exists(project.database, schema)
+        assert not exists
+
+
+def test__postgres__specific_model(project):
```

Comment:
The 'postgres' piece should be removed from all of the test names. This one, for example, should be 'test_specific_model'.